src/cable/udp/cable_udp_obj.cpp

```diff
@@ -3,14 +3,27 @@
 #include "../asio_ioservice_manager.hpp"
 #include <libs0.hpp>
 
+#define ANTINET_socket_use_two_and_reuse
+
 using namespace boost::asio::ip;
 
-c_cable_udp::c_cable_udp(shared_ptr<c_asioservice_manager> iomanager)
+c_cable_udp::c_cable_udp(shared_ptr<c_asioservice_manager> iomanager, const c_card_selector &source_addr)
```

Comment:
yes, that's a better for performance

(though it can't be const ptr &   of course, as this constructor will keep ownership of this shared object)